(Reviewed) Proofreading after tutorial duel

John-hash-tech opened the pull request:
This includes anything I encountered between the end of the tutorial conversation with Master Konoha and the beginning of the guild tutorial with Leaf. There are lots of small changes, including a few card names that didn't appear to be correct.

John-hash-tech committed:
Update 70002.txt

"Monsters" is plural. "Succubi" is plural. Punctuation goes inside quotation marks in American English.

John-hash-tech committed:
Temp full commit after tutorial

22/26/40003: changes to card names
22: changed Machete to Nodachi, since I think that's what weapon he's holding. "Giant Machete" is sort of contradictory anyway.
26: changed Lightning to Windcall Summoner, since Lightning appears to be an error. I am open to feedback on the new translation for this card name.
40003: changed Sinkhole, but not sure how translators would like this card to be named, so I left a few prospective options

10005, 10006, 10007, 10008: changes to slime monster dialog
10005: on line 136, I initially used "his willpower," but figured "their willpower" was a more inclusive alternative. Line 211 is too long, but none of the text actually spills off-screen, so probably fine.
10006: On line 41, I replaced an em-dash with a comma. I'm not opposed to em-dashes, but people rarely use them and seem to associate them with AI output.
10007: on line 187, I changed "succubi's" to "succubus's." If I'm understanding correctly, monster cards do not typically count as succubi. Therefore, the Slime Succubus is operating independently of other succubi, so we should use a singular noun here. The card Demon Slayer supports my case because he does double damage to Slime Succubus and not her monsters. Line 189 is sort of the opposite, where I interpreted it as him serving the general interests of all succubi.
10008: the text change in line 33 sounds more natural to me. Line 42 was untranslated in game.

70003: Slime Succubus dialog changes. I marked lines 25 and 36 for seemingly changing point of view. I would prefer to make those lines more clearly first-person. The second half of line 127 seems like a mistranslation. Lines 122-124 are untranslated.

80017: lines 8 and 42 are just word changes that sound more natural to me.

John-hash-tech committed:
Update 70002.txt

I don't think the word "now" should be used on line 27, since the pattern has been developing over the course of several years.

John-hash-tech committed:
Update 70003.txt

The guild master in Morgen is called Leaf in other files. Changing from Lief.

John-hash-tech committed:
Comment Cleanup

Removed comments I left for myself from the files I worked on.

John-hash-tech commented:
Notes:
10005.txt: line 211 overflows textbox
10008.txt: line 42 untranslated
70003.txt: lines 122-124 partially untranslated

Giratena commented:
Marking as ready for review as discussed on Discord ^^

HotHeadEmperor commented on CSV/Language/English/Card/10005.txt:
```diff
@@ -127,13 +127,13 @@ EU5_thighJobAnime,
 あは、おねだりするみたいなその目❤射精したくてたまらないんだ。,Aha! Aw; don't give me those puppy-dog eyes.❤ You're really dying to ejaculate; huh?
 ううっ、強い…。このままじゃ負けちゃう…。,Ough! You're pretty strong... At this rate I'll lose...
-あのね、わたしちょっと特技があって、やられる前に試してもいいかな・・・？,Um hey; I was wondering. Could I try one last little skill before you go and beat me up...?
+あのね、わたしちょっと特技があって、やられる前に試してもいいかな・・・？,Um; hey; I was wondering. Could I try one last little skill before you go and beat me up...?
```
Good catch here, but if you don't mind going a little further, it might actually flow better if you put an ellipses after 'wondering,' What do you think?

HotHeadEmperor commented on CSV/Language/English/Card/10005.txt:
```diff
@@ -224,5 +224,5 @@ EU5_thighJobAnime,
 さいごまで幸せそうな顔だったね。こんな風に終われるなら、十分いい終わり方でしょ？,You looked happy until the very end~ But I guess if your life ended in this way; it's probably not a bad way to go out; huh?
 こうして主人公は、最初の戦闘でスライムのモンスターに負け、命が干からびるまで精を吸い取られた。,And so; the protaganist was defeated in his first battle by a slime monster; who milked his dick until his life force faded.
-妹を探すはずだった冒険は、あっけなく下級モンスターの手によってその幕を閉じ、,The adventure of looking for his sister upbruptly ended; by the hand of a lesser monster.
+妹を探すはずだった冒険は、あっけなく下級モンスターの手によってその幕を閉じ、,The adventure he started to find his sister was ended abruptly at the hand of a lesser monster.
```
Good catch, but what do you think of 'came to an abrupt end'?

HotHeadEmperor commented on CSV/Language/English/Card/10006.txt:
```diff
@@ -52,20 +52,20 @@ slime_superBigBreastPaizuri,
+くすくす❤興奮しすぎ。そんなに早くしごいてたらあっという間に負けちゃうよ。,*Giggles*~❤ You're getting too excited. If you keep stroking that fast; you'll lose before you know it!
 今ならまだ間に合う・・・❤手を止めて武器をとれば、まだ戦える・・・❤ほら、がんばって♪,It's not too late yet...❤ If you stop and grab your weapon; you can still fight...❤ Come on; you can do it♪
-わたしもたくさん、おっぱい揺らして応援してあげるから♪えいえい❤,I'll shake my boobs a lot to cheer you on too♪ Go; go~❤
+わたしもたくさん、おっぱい揺らして応援してあげるから♪えいえい❤,I'll shake my boobs a lot to cheer you on; too♪ Go; go~❤
```
Good catch, but since you're already here, would you add a ~ before the music note as well?

John-hash-tech replied:
I'm thinking of changing the "Go, go~❤" at the end of line 57 into a "Go! Go!❤" instead. There's just so much flirtatious dialog already that using exclamation points here might be an opportunity to make this line stand out a little. I'm all for consistency, but variety will help keep readers engaged. I am open to all opinions.

HotHeadEmperor replied:
I think that would be a good call.

HotHeadEmperor commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -48,11 +48,11 @@ EU7_paizuriAnime,
 あは、興奮しすぎ❤谷間を作ってあげただけで、そんなにガン見しちゃって・・・。恥ずかしくないのかな？他の仲間に笑われちゃうよ？,Aha! You're way too horny right now.❤ I only squeezed my boobs together; and now you're ogling them so intently... Have you no shame? Won't your allies laugh at you for doing such a thing?
 そんなこともうどうでもいっか。おっぱいのことだけしか頭になくて、とっても間抜けなお顔でおちんちん扱いてる❤,Not that I mind; of course. Empty your mind of everything except boobs; and stroke your dick with that stupid look on your face.❤
-しこしこ、しこしこって・・・❤気持ちよさそうだねー。変態で、最高にカッコ悪いよ❤新米マスターさん❤,Stroke your dick- Pump it and rub it...❤ Looks like you're feeling good~ Perverts really do look stupid huh? Newbie master~❤
+しこしこ、しこしこって・・・❤気持ちよさそうだねー。変態で、最高にカッコ悪いよ❤新米マスターさん❤,Stroke your dick- Pump it and rub it...❤ Looks like you're feeling good~ Perverts really do look stupid; huh? Newbie master~❤
```
Good catch. Couple things to add. Maybe change the - to a . and reword that last line to "Perverts _like you_ really do look stupid"

What do you think?

John-hash-tech replied:
Great stuff!

HotHeadEmperor commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -48,11 +48,11 @@ EU7_paizuriAnime,
 ふふ、馬鹿にされて興奮しちゃった？しごくの、もっと早くなってるじゃん❤,Fufu; did it turn you on when I made fun of you? You're stroking even faster now~❤
 敵の目の前でくすくす笑われて、悔しいはずなのに、おちんちんをいじる手がとまらない・・・❤,It should be making you feel frustrated when your enemy is laughing at you like this; and yet you can't stop playing with yourself...❤
-いいよ。ほら、おっぱいを顔の前でたぷたぷしてあげるから・・・❤情けない顔のまま、射精しちゃえ。,It's okay. Hora;  I'll bounce my boobs in front of your face...❤ Hold that pathetic expression...and shoot it all out~.
+いいよ。ほら、おっぱいを顔の前でたぷたぷしてあげるから・・・❤情けない顔のまま、射精しちゃえ。,It's okay. Hora;  I'll bounce my boobs in front of your face...❤ Hold that pathetic expression...and shoot it all out~
```
Great catch, but could you change 'hora' to 'here' for me?

John-hash-tech replied:
I asked about the use of hora and ara in the translation-help-general channel because I had the same thought. Giratena's answer was that it's sort of like vernacular used by succubi and monster girls. If that's done consistently throughout the project, I don't mind leaving it here, as this file is for Busty Slime Girl.

HotHeadEmperor replied:
It really isn't that much. Like if we wanted it to be a character trait on a couple charcters we could, but I dont know that it would fit this particular charcters, you know?

John-hash-tech replied:
Ok, I'll make changes as I go whenever I encounter hora or ara.

HotHeadEmperor commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -112,9 +112,9 @@ EU7_paizuriAnime,
 あと一歩まで追い詰めたのに、あなたも結局、モンスターと一緒だったね。おっぱいに負けたかったんでしょ？新米マスターさん❤,Oh Newbie Master♪You were so close to beating me; but in the end you lost to my boobs like the rest of the other monsters; huh? Newbie Master~❤
 みんなしてマスターの命令に逆らって・・・。くすくす❤おっぱいに弱すぎだよ❤,You went and disobeyed your master just like the others... Kuhuhu.❤ You have a serious weakness when it comes to boobs.❤
-ほら見て。谷間がどろどろ❤わたしのおっぱい、オカズにできてよかったね。新米マスターさん❤,Look at this. My cleavage is all slimy.❤ You really enjoyed using my boobs as fap material huh; Mr. Newbie Master?❤
+ほら見て。谷間がどろどろ❤わたしのおっぱい、オカズにできてよかったね。新米マスターさん❤,Look at this. My cleavage is all slimy.❤ You really enjoyed using my boobs as fap material; huh; Mr. Newbie Master?❤
```
Since a comma before the 'huh' is usually seen as a stylistic choice and not a rule of grammar, I suggest not using it to avoid the double comma here.

HotHeadEmperor commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -151,7 +151,7 @@ EU7_paizuriAnime,
 そっかー、残念だなー❤\nでも、ほら、見てぇ❤,Is that so~ What a pity~❤ But; look at this; hora~❤
 あなたを二回もイかせちゃったおっぱい❤\n挟んで、ずりずりされて、\nとってもきもちよかったよねー❤,These are the boobs that made you cum twice.❤ It felt really good when I rubbed your dick between them; right?❤
-もうわたしのおっぱいは飽きちゃったかなぁ❤\nあの腰が溶けちゃうみたいなパイズリは、\n二度と味わえなくなっちゃうんだよ？,Are you sure you've had enough of my boobs...?❤ If you pass this up; you'll never taste my mind melting dick melting paizuri ever again; you know?
+もうわたしのおっぱいは飽きちゃったかなぁ❤\nあの腰が溶けちゃうみたいなパイズリは、\n二度と味わえなくなっちゃうんだよ？,Are you sure you've had enough of my boobs...?❤ If you pass this up; you'll never taste my mind melting; dick melting paizuri ever again; you know?
```
Good catch, but could you change it to 'mind-melting' and 'dick-melting' for me?

Giratena replied:
I think this is a good suggestion!

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -39,7 +39,7 @@ EU8_handJobAnime,
 さあ、わたしに屈服しなさい。天国を見せてあげる・・・❤,Now; give in to me. I'll show you what heaven feels like...❤
 彼女を振り払い、とどめを刺した,Shake her off and deal the final blow.
-体を動かすことができなかつた,Stay still.
+体を動かすことができなかつた,Stay still. 
```
Looks like you just added a space here on accident.

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -83,13 +83,13 @@ EU8_handJobAnime,
+おちんちんびくびくして、もう限界なんだ❤\n好きなだけ出すといいわ❤\n仲間のことなんて忘れて、どぴゅどぴゅってイっちゃえ、マゾ男❤,Your little weenie is twitching. You're already at your limit; aren't you?❤ Go on; cum as much as you want~❤ Forget about your friends and just splurt it all out; mister masochist~❤
 あん❤どばどば出てる❤エナジーおいしいー❤,Ah~❤ It's spilling out~❤ Your energy is so delicious~❤
-ふふ、中だしできなくて残念だったわね。\nこれでわたしたちの勝ち。\n自分から自爆しちゃうカードマスターで助かったなー♪,Hehe; too bad you couldn't finish inside. Guess this means we win. Lucky for us; the cardmaster ended up screwing himself over♪
+ふふ、中だしできなくて残念だったわね。\nこれでわたしたちの勝ち。\n自分から自爆しちゃうカードマスターで助かったなー♪,Hehe; too bad you couldn't finish inside. Guess this means we win. Lucky for us; the Cardmaster ended up screwing himself over♪
```
Cardmaster is being used as a pronoun not a proper noun, so lowercase was correct.

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -83,13 +83,13 @@ EU8_handJobAnime,
 おまんこに入れなくなたって、イっちゃうんでしょ？\n馬鹿にされて喜ぶ、ロリコンマゾだもんね❤,Even if you can't get it inside me; you'll still come; won't you? You're a lolicon masochist who loves being mocked.❤
 ほら、シコシコって言われるだけで手が動いちゃう❤\nもっと気持ちいいことしてもらえるかもしれないのに、\nシコシコ❤ シコシコ❤ やめられない❤,Look; just hearing 'fap~ fap~' makes your hand start moving on its own~❤ You could be feeling something even better; but you just can't stop; can you~?❤ Fap~❤ You just can't help yourself.❤
-おちんちんびくびくして、もう限界なんだ❤\n好きなだけ出すといいわ❤\n仲間のことなんて忘れて、どぴゅどぴゅってイっちゃえ、マゾ男❤,Your little weenie is twitching. You're already at your limit; aren't you?❤ Go on; cum as much as you want~❤ Forget about your friends and just splurt it all out; you masochistic mister~❤
+おちんちんびくびくして、もう限界なんだ❤\n好きなだけ出すといいわ❤\n仲間のことなんて忘れて、どぴゅどぴゅってイっちゃえ、マゾ男❤,Your little weenie is twitching. You're already at your limit; aren't you?❤ Go on; cum as much as you want~❤ Forget about your friends and just splurt it all out; mister masochist~❤
```
If you want to flip this like this, 'Mister Masochist' would be a substitute for a name, so it would be a proper now and need to be capitalized. I think it's a good idea to change it like this, so I would definitely recommend doing so.

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -83,13 +83,13 @@ EU8_handJobAnime,
 もしかしたら、もっと気持ちいいサービスしちゃうかも。\nほら、おいで、おいで・・・❤,Maybe I'll even give you something more pleasurable~ Come on; come over here...❤
 きゃはは❤ ほんとにまたオナニーしちゃうんだ。\n武器を放りだして敵の女の子の前でシコシコ、シコシコ・・・❤,Kya-haha!❤ You're really going to do it again? Throw away your weapon and start fapping in front of the enemy girl~ fap fap; fap fap...❤
-わたしの体も罪だなぁ❤ せくしー過ぎてこまっちゃう。\nほら、近くでたっぷり見ていいよ❤ くすくす❤,My body's a sin~❤ It's so sexy it's troublesome.. Come on; you can admire my body as much as you want.❤ *Giggle*❤
+わたしの体も罪だなぁ❤ せくしー過ぎてこまっちゃう。\nほら、近くでたっぷり見ていいよ❤ くすくす❤,My body's a sin~❤ It's so sexy you don't even know what to do... Come on; you can admire my body as much as you want.❤ *Giggle*❤
```
VERY good rewording. I like this a lot.

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -110,15 +110,15 @@ EU8_handJobAnime,
 ううっ…。\nやめろ、その人から離れろ・・・！,Ugh... Stop; get away from him...!
-あら。健気な護衛さんですね❤\nマスターのお兄さんはまだ頑張るつもりですか？,Oh my. How admirable of you; protector~❤ Is my big brother; the Master; still planning to give it his all?
-けど今さら無駄です❤\nお兄さんが負けちゃったせいで、\nみ～んなわたしたちに食べられちゃうんですよ❤,But it's pointless now.❤ Because of you lost; everyone will end up being eaten by us~❤
+あら。健気な護衛さんですね❤\nマスターのお兄さんはまだ頑張るつもりですか？,Oh my. How admirable of you; protector~❤ Is my big brother; the cardmaster; still planning to give it his all?
```
Good catch here, but I think we can go further. I don't know that we need those commas. What do you think?

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -110,15 +110,15 @@ EU8_handJobAnime,
 くそぅ・・・。,Damn...
 僕は無意味とわかっていながらも商人の元へと這い寄ろうとする。しかし、その動きは途中で止められてしまう。,I crawled toward the merchant; even knowing it was meaningless. However; my movement was suddenly stopped.
-見ると、バトルで僕を倒したメルティスライムが目の前に立ちふさがり、僕をにやにやとした顔で見下ろしていた。,When I looked up; the Melty Slime; who had defeated me in battle stood in my way; grinning down at me.
+見ると、バトルで僕を倒したメルティスライムが目の前に立ちふさがり、僕をにやにやとした顔で見下ろしていた。,When I looked up; the Melty Slime; who had defeated me in battle; stood in my way; grinning down at me.
```
I think Gira would recommend an em dash here:

the Melty Slime -who had defeated me in battle- stood in my way; grinning down at me.

John-hash-tech replied:
I think em dashes are easily misused and often give the impression that the writer is trying to show off, so I typically steer clear of them, but this is the perfect spot for some em dashes.
Edit: I double checked and we are using the em dash character consistently throughout the project, with no occurrences of the en dash character.

HotHeadEmperor commented on CSV/Language/English/Card/10008.txt:
```diff
@@ -177,6 +177,6 @@ EU8_handJobAnime,
 スライムオナホでぐちゅぐちゅされると\n何度でもおちんちんイっちゃう❤\nかわいい、かわいい素直なチンポ❤,When the slime pussy squishes you; your weenie just keeps cumming over and over~❤ Such a cute; obedient little weenie~❤
 ほら、わたしの手にもっとエナジー吐き出して❤\nロリコンチンポ、ぐちゅぐちゅ犯されて、\nあんたの命を、全部全部、射精するの❤,Come on; spill more of that energy into my hand~❤ Such a cute little weenie; teased; squished; and toyed with~ I'll make you spill every last drop of your life~❤
-きゃはは❤\n止まらないね。イっちゃうね❤\n逝って終わっちゃうね❤ほら、イケ❤マーゾ❤,Kyahaha~❤ Can't stop; right~? You're gonna cum for me; huh~❤ You're almost there; huh~?❤Go on; cum for me~❤ Masochist~❤
+きゃはは❤\n止まらないね。イっちゃうね❤\n逝って終わっちゃうね❤ほら、イケ❤マーゾ❤,Kyahaha~❤ Can't stop; right~? You're gonna cum for me; huh~?❤ You're almost there; huh~?❤Go on; cum for me~❤ Masochist~❤
```
Good catch, but ~ counts as punctuation, so you'd want to remove that to add the question mark. Also, would you put a space before 'Go on' while you're at it?

John-hash-tech replied:
Would you mind if I kept the question marks instead? My impression is that Melty Slime is big on the teasing- and humiliation-type appeal. I like the idea of her asking these pointed questions as if to make the protagonist say it himself, then bringing the hammer down with the masochist accusation.

HotHeadEmperor replied:
Sorry that's what I was trying I suggest was delete the ~ to keep the ? because i do agree it works better

John-hash-tech replied:
Sorry, I didn't read your first comment carefully enough. Glad we agree here!

HotHeadEmperor commented on CSV/Language/English/Card/80017.txt:
```diff
@@ -5,11 +5,11 @@ default
 路地裏の入り口で、一人の女の子がつまらなさそうに壁にもたれかかって立っていた。,I saw a bored-looking girl leaning against a wall at the side of the street.
 彼女は僕を見つけると、ニコリと笑い、こちらに話しかけてくる。,When she noticed my gaze; she smiled and came over to me.
-ねえ、キミ。この街ははじめて？\nきょろきょろしながら歩いて、田舎から出てきたばっかりなのかな？\nくすくす♪,Hey; is this your first time in a city? You're looking like a country bumpkin with the way you're looking around. Ehehe♪
+ねえ、キミ。この街ははじめて？\nきょろきょろしながら歩いて、田舎から出てきたばっかりなのかな？\nくすくす♪,Hey; is this your first time in a city? You're looking like a country bumpkin with the way you're gawking. Ehehe♪
```
Really good word choice here. Love it.

John-hash-tech replied:
Thanks! I'm planning to change it to "gawking at everything" to make it clear that he's gawking at all the sights of the city and not just leering at her.

HotHeadEmperor commented on CSV/Language/English/Card/70003.txt:
```diff
@@ -17,12 +17,12 @@ Kiss01,
 短い間ですが、よろしくお願いします。,It might only be for a short time, but I'll be counting on you.
 一緒に来てくださいますか！ありがとうございます。,So; you'll come with me! Thanks a bunch.
-短い間ですが、よろしくお願いしますね！,Then for the time being; I look forward to working with you!
+短い間ですが、よろしくお願いしますね！,Then; for the time being; I look forward to working with you!
```
I don't know that the comma is necessary here.

John-hash-tech replied:
I'm really divided on this one. On one hand, the speaker isn't necessarily going to pause where the new comma is. On the other hand, you could rewrite the sentence without it and the result would still be a complete sentence:
`Then I look forward to working with you!`

Since it's just an extra phrase being spliced in, I think we should use commas to set it apart. I don't think it's far enough removed from the topic to merit em dashes or parentheses either.

[Based on this source,](https://owl.purdue.edu/owl/general_writing/punctuation/independent_and_dependent_clauses/index.html) we're either dealing with case 2 or case 3, and that's the crux of the problem.

Giratena replied:
I'm happy to leave as is. I can see both sides but both are valid I think ^^

HotHeadEmperor commented on CSV/Language/English/Card/70003.txt:
```diff
@@ -56,22 +56,22 @@ Kiss01,
+なぜだろう、嫌な予感がする。見た目はただの小さな女の子のはずなのに・・・。,I wasn't sure why; but I had a bad feeling about this girl. She looked like your typical cute girl but...
+んー・・・。何か怪しまれるようなところあったかなぁ・・・。あ、もしかして、,Hnnn...Why are you so suspicious...? Ah; could it be...
+魔力が漏れているのがバレちゃいました？くすくす❤,You can sense my magic aura? Teeheehee~❤ 
+気づかずに来てくれれば、唇でちゅってしてあげたのに・・・❤ほら、こんな風に。,If you had only come over here; I would have smooched you on the lips.❤ Hora; just like this...
```
Good change. Wanna take a stab at changing that 'hora'?

John-hash-tech replied:
Same as above, but this file is for Succubus Girl (the slime girl cardmaster). I especially like this one because it's the first time she says "hora" and this line is immediately after her succubus wings become visible.

Giratena replied:
Yep Hora is good here!

HotHeadEmperor reviewed (commented):
Really good stuff. Most of what I left is actually asking you to go a step further, so I definitely think you're moving things in the right direction!

John-hash-tech committed:
Update 10003.txt

Removed the line break characters from the English text in the file for the Samurai Master Shizuku monster card

John-hash-tech committed:
Update 10001.txt

Changed multiple instances of "in my breasts" to "between my breasts." I did not take context into account and will have to retest the encounter to make sure the changes don't look out of place. I also left the instance in line 58 unchanged because I like the way it's written already.

John-hash-tech committed:
Update 80017.txt

Updated a line in the English translation for the massage tout in the Morgen entrance area. With this change, it should be clearer that the main character is gawking at the entire city, not just the girl.

John-hash-tech commented:
I notice there are no comments on the three card names I changed. Does anyone have any objections? Shukuchi in particular could be more localized, but I'm not sure exactly what would be ideal. "Flash Step," "Distance Compression," and "Bypass" all seem reasonable. I also think just going with "Shukuchi" is fine tbh.

Giratena commented:
> I notice there are no comments on the three card names I changed. Does anyone have any objections? Shukuchi in particular could be more localized, but I'm not sure exactly what would be ideal. "Flash Step," "Distance Compression," and "Bypass" all seem reasonable. I also think just going with "Shukuchi" is fine tbh.

I haven't looked at this PR yet in detail (possibly after Hothead's comments have all been resolved), but from a quick glance I do like the changes you've done with the card names. Having a name like "Shukuchi" is also very yugioh like, so I'm all for it ^^

John-hash-tech committed:
PR feedback part 1

Based on PR feedback, these are the changes I am 100% in favor of making. I will make a separate commit for the other changes I have in mind.

John-hash-tech committed:
PR feedback part 2

These are changes based on PR feedback that might need to be discussed further.

John-hash-tech committed:
Hora conversion and ellipsis sweep

Converted all instances of "hora" to English words or phrases. Also did another pass on ellipsis that have more or less than three periods or that use special characters. Also also, removed a few instances of double spaces and added commas where they belong.

John-hash-tech commented:
I'll be ready for review again after one more round of playtesting. Upcoming changes will include a rewrite of every "hora" into English.

John-hash-tech committed:
Update 70003.txt

Changed a comma to a semicolon so it can properly serve as an escape character.

John-hash-tech committed:
Update 10006.txt

Various small changes to Huge Boobs Slimegirl

John-hash-tech committed:
Update 10006.txt

Small sentence change for huge Boobs Slimegirl

John-hash-tech committed:
Update 10007.txt

Updated for consistency with the change in 10006.txt

John-hash-tech committed:
Update 10005.txt

Normally, we use asterisks to indicate a sound effect (e.g. *giggle*). Here, they seem to indicate some type of emphasis. I think we should use either italics or bold instead. Removing the asterisks entirely is another option.

John-hash-tech commented:
Ok, ready for review again!

Giratena commented on CSV/Language/English/Card/10005.txt:
```diff
@@ -3,17 +3,17 @@ default,
 　スライムの下級モンスターであり、ステータスは低水準。しかし、彼女たちの粘液には相手を敏感にさせる成分が含まれており、油断するとステータスからは想像できないほどの強い快感を与えられてしまうので注意が必要である。下級モンスターとはいえ、スライムまみれにされた男性器をぐちゅぐちゅにされれば、多くの男はひとたまりもない。,A lesser slime with low stats. However; the viscus liquid produced by their body contain compontents that can make their foes incredibly sensitive. Caution is heavily advised as if one is not careful; the pleasures they give can vastly outw ...
 ご主人さまにメロメロみたいだねぇ❤くすくす❤,Seems my mistress has you head over heels for her; huh?❤ *Giggle*❤
-ほらほら、ぬるぬるの手でシコシコしてあげるから、ご主人さまのこと大好きって思いながら気持ちよくなっちゃえ。,Hora hora. I'll stroke you with my slippery hand so you can feel good while you indulge in your loving fantasies about her.
+ほらほら、ぬるぬるの手でシコシコしてあげるから、ご主人さまのこと大好きって思いながら気持ちよくなっちゃえ。,Here, here. I'll stroke you with my slippery hand so you can feel good while you indulge in your loving fantasies about her.
```
Hmm I dont think "here here" flows very well. There's also a ,

Maybe try something else otherwise hora hora isnt bad either.

Giratena commented on CSV/Language/English/Card/10005.txt:
```diff
@@ -22,38 +22,38 @@ EU5_thighJobAnime,
-むちむちのふとももで挟んであげる。ほら、ぱっくん❤,I'll insert your dick between my plump thighs. Hora. Squish!❤
-むぎゅーって圧迫されてたまらないよね？ぬるぬるになったおちんちん、このふとももの間でたくさんいじめてあげるね❤ほらほら❤,The pressure you feel from being squeezed is irresistable; right? I'll tease your slippery penis lots between my thighs; OK?❤ Hora hora~❤
+むちむちのふとももで挟んであげる。ほら、ぱっくん❤,I'll insert your dick between my plump thighs. There. Squish!❤
+むぎゅーって圧迫されてたまらないよね？ぬるぬるになったおちんちん、このふとももの間でたくさんいじめてあげるね❤ほらほら❤,The pressure you feel from being squeezed is irresistable; right? I'll tease your slippery penis lots between my thighs; OK?❤ Here, come on~❤
```
, here

Giratena commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -4,37 +4,37 @@ EU7_paizuriAnime,
 おっぱいが好きなあなたなら、\nこんな大きいおっぱいでいじめてもらえて、うれしいよねー❤\nお口からよだれがでちゃってアヘアへ状態❤,A big boobie lover like yourself must be overjoyed to have their dick teased by boobs as large as mine; huh?❤ Your're already moaning and even drooling a little.❤
-ぷるんぷるんのスライムおっぱいで、もっとズリズリして\n骨抜きにしてあげる❤\nほらほらほら・・・❤,I'll rub you even more between my bouncy slime boobies and make you fall deeper under my spell.❤ Hora hora hora...❤
-ほら、とびっきりのおっぱいで挟んで昇天させてあげる❤,Hora; I'll wrap my superior boobs around your dick and send you to heaven.❤
+ぷるんぷるんのスライムおっぱいで、もっとズリズリして\n骨抜きにしてあげる❤\nほらほらほら・・・❤,I'll rub you even more between my bouncy slime boobies and make you fall deeper under my spell.❤ Come on, come on, come on...❤
```
Watch the , here. They can break the translation

Giratena commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -4,37 +4,37 @@ EU7_paizuriAnime,
-ほらぁ、むぎゅっておちんちん挟み込んで、\nずりずり、ぐちゅぐちゅ❤\nぷるんぷるんのおっぱいで死ぬほど気持ちよくされちゃう❤,Hora~ I’ll squeeze your dick between my boobs and... *Slide* *squish*...❤ It feels so so good you can feel your consciousness slipping away; right?❤
-頭の中とろとろにされて、戦えなくされちゃうね❤\nおっぱいに負けてエナジーわたしにちょうだぁい❤\nほらほらほら・・・❤,There's no fighting back anymore with such an empty mind.❤ Just give into my boobs and give me your mana.❤ Hora hora hora...❤
+ほらぁ、むぎゅっておちんちん挟み込んで、\nずりずり、ぐちゅぐちゅ❤\nぷるんぷるんのおっぱいで死ぬほど気持ちよくされちゃう❤,Here~ I’ll squeeze your dick between my boobs and... *Slide* *squish*...❤ It feels so; so good you can feel your consciousness slipping away; right?❤
+頭の中とろとろにされて、戦えなくされちゃうね❤\nおっぱいに負けてエナジーわたしにちょうだぁい❤\nほらほらほら・・・❤,There's no fighting back anymore with such an empty mind.❤ Just give in to my boobs and give me your mana.❤ Come on, come on, come on...❤
```
Here too

Giratena commented on CSV/Language/English/Card/10007.txt:
```diff
@@ -176,9 +176,9 @@ EU7_paizuriAnime,
 やりたいこともやらなきゃいけないことも、\nぜーんぶおっぱいに塗り替えられちゃったね❤\nくす❤くすくす❤,All those things are getting replaced with the urge to shoot your load between my boobs.❤ Kuhu~❤ Kuhuhu!❤
 あう、あうううっ❤,Ahh; aooohhh~❤
-ほらぁ、カードマスターとしての人生に、バイバイしちゃお？\nおっぱいにどぴゅどぴゅーって射精して、\n今日からキミはサキュバスのペットだよ❤,Hora; time to say "bye bye!" to your life as a cardmaster; OK? Cum in my boobs; and turn yourself into a succubus' pet.❤
+ほらぁ、カードマスターとしての人生に、バイバイしちゃお？\nおっぱいにどぴゅどぴゅーって射精して、\n今日からキミはサキュバスのペットだよ❤,Weeell, shall we say "bye bye!" to your life as a cardmaster? Cum in my boobs; and turn yourself into a succubus's pet.❤
```
, again

Giratena reviewed (changes requested):
Ok, had a look through and spotted some things! Once the comments from myself and Hothead are resolved we can probably get this merged in ^^

John-hash-tech committed:
PR feedback

Mostly cleaning up unescaped commas. Also one retranslation of a horahora.

John-hash-tech commented:
I fixed the unescaped commas. 

While doing that, I found out that you can sometimes leave them as commas in the file and have them show up as commas in game, but only in black popups that appear during the tutorial, and not consistently. I just wanted to note that here since I made an issue about the semicolons we have showing up in the tutorial.

Giratena commented:
Looks like you have some merge conflicts. I'll merge it in after they're resolved!

Giratena commented:
> I fixed the unescaped commas.
> 
> While doing that, I found out that you can sometimes leave them as commas in the file and have them show up as commas in game, but only in black popups that appear during the tutorial, and not consistently. I just wanted to note that here since I made an issue about the semicolons we have showing up in the tutorial.

You are absolutely correct! As a downside it means we can't have commas in those black tutorial prompts either, which sucks.

I've let Iroji know about it. Hopefully he has time to fix it whenever he is free ^^

John-hash-tech committed:
Merge branch 'develop' into Proofreading-After-Tutorial-Duel

Giratena reviewed (approved):
Looks good! Nice work :D
